.github/workflows/ci.yml

```diff
@@ -67,12 +67,6 @@ jobs:
           - name: Windows
```

Comment:
Maybe also do

```suggestion
          - name: Win
```


To work around GitHub UI issues

![image](https://user-images.githubusercontent.com/204609/162410053-738893f6-5372-4891-adfd-9e15664fbe27.png)